src/index.js

```diff
@@ -70,7 +70,12 @@ function addPackageData (inputs, path) {
 function computeHash (pepper, paths, seed) {
   const inputs = []
   if (pepper) inputs.push(pepper)
-  if (seed) inputs.push(seed)
+  if (seed) {
```

Comment:
I should probably rename `seed` to `salt`. Was going back and forth on that before.